core/client/fs/src/main/java/alluxio/client/file/cache/CacheRestorationCallback.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.file.cache;
+
+public interface CacheRestorationCallback {
+  /**
+   * The callback that will be invoked when the page store finishes restoring pages after start up.
+   */
+  void onCacheRestorationSuccess();
+
+  static void NO_OP() {
+    //NOOP
+  }
```

Comment:
You can add a static field on the interface:

```suggestion
  CacheRestorationCallback NOOP = () -> { };
```